steps/steps-app-gallery-reports/0.9.0/step.yml

```diff
@@ -0,0 +1,245 @@
+title: Huawei AppGallery Connect Reports
+summary: Gets the different Reports from AppGallery Connect for a specific published
+  app.
+description: |-
+  Gets the next reports from AppGallery Connect:
+  - Download and Installation Report
+  - In-app Payment Report
+  - Paid Download Report
+  - Installation Failure Data Report
+
+  This step is based on [AppGallery Connect Reports](https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-References/agcapi-appdownloadexport)
+website: https://github.com/ferPrieto/steps-app-gallery-reports
+source_code_url: https://github.com/ferPrieto/steps-app-gallery-reports
+support_url: https://github.com/ferPrieto/steps-app-gallery-reports/issues
+published_at: 2021-03-06T00:13:45.025487Z
+source:
+  git: https://github.com/ferPrieto/steps-app-gallery-reports.git
+  commit: 6c74a96cdf30aa2ce82f753ae6f33a702e644916
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- xamarin
+- react-native
+- flutter
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: jq
+  apt_get:
+  - name: jq
+is_requires_admin_user: true
+is_always_run: true
+is_skippable: false
+run_if: ""
+inputs:
+- huawei_app_id: null
+  opts:
+    category: Required Config
+    description: App ID can be found in the `App information` section in App Gallery
+      Connect.
+    is_expand: true
+    is_required: true
+    title: App ID
+- huawei_client_id: null
+  opts:
+    category: Required Config
+    description: Client ID generated in `AppGallery Connect -> Users and permissions
+      -> API key -> Connect API -> API client`
+    is_expand: true
+    is_required: true
+    title: Client ID
+- huawei_client_secret: null
+  opts:
+    category: Required Config
+    description: Key Secret generated in `AppGallery Connect -> Users and permissions
+      -> API key -> Connect API -> API client`
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: Key
+- download_installation_language: en-US
+  opts:
+    category: Download and Installation
+    description: Query language. The report title is displayed in different languages.
+      Only simplified Chinese (zh-CN), English (en-US), and Russian (ru-RU) are supported.
```

Comment:
```suggestion
      Only simplified Chinese (zh-CN), US English (en-US), and Russian (ru-RU) are supported.
```

Reply:
This is part of the official Huawei documentation, therefore this addition you pointed is ok, but not part of the official description of the parameter. Thanks though